plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueSnapshot.java

```diff
@@ -157,6 +157,11 @@ public boolean isClosed() {
     return closed.get();
   }
 
+  @Override
+  public boolean isPersistent() {
```

Comment:
I think all of the isPersistent() methods are no longer necessary for this implementation.  it is useful information to have, but it doesn't serve a purpose currently.  probably should remove